docs/standard/microservices-architecture/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -78,7 +85,7 @@ public async Task<IActionResult> UpdateProduct([FromBody]CatalogItem product)
 }
 ```
 
-In this case, since the origin microservice is a simple CRUD microservice, that code is placed right into a Web API controller. In more advanced microservices, it could be implemented in the CommandHandler class, right after the original data is committed.
+In this case, since the origin microservice is a simple CRUD microservice, that code is placed right into a Web API controller. In more advanced microservices, like when using CQRS approaches, it could be implemented in the CommandHandler class, right after the original data is committed.
```

Comment:
The phrase "right after the original data is committed." is confusing. I think you mean that the code would be executed right after the original data is committed.

If so, I'd split this into two sentences. The second would be "The new code in the `CommandHandler` class executes right after the original data is committed."

Reply:
👍 - OK, I chaged it to:
In this case, since the origin microservice is a simple CRUD microservice, that code is placed right into a Web API controller. 
In more advanced microservices, like when using CQRS approaches, it can be implemented in the CommandHandler class, within the Handle() method.